ChainSafe/web3.unity
# Web3auth seperate window for webgl

Web3auth now opens in a seperate window for webgl so that it doesn't close the game.

Closes #894 

To test:
Login in with web3auth in webgl.

Expected behavior:
A new window will open to let you connect.
